appveyor.yml

```diff
@@ -1,18 +1,20 @@
 image: Visual Studio 2019
 
+version: '{build}'
+
 # Download script file from GitHub
 init:
-  ps: |
-    $ErrorActionPreference = "Stop"
-    Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
-    Import-Module '..\appveyor-tool.ps1'
+  - ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"

```

Comment:
@Yuri05 Unrelated but ensure that we are using the global version (suite component)